src/Tasks/RoslynCodeTaskFactory/RoslynCodeTaskFactory.cs

```diff
@@ -113,7 +113,6 @@ public sealed class RoslynCodeTaskFactory : ITaskFactory
         /// <inheritdoc cref="ITaskFactory.FactoryName"/>
         public string FactoryName => "Roslyn Code Task Factory";
 
-        /// <inheritdoc />
```

Comment:
📝 This was inheriting from https://github.com/dotnet/msbuild/blob/40beb5a025fb18c82a159f407b5ce2383531a8eb/src/Framework/ITaskFactory.cs#L20-L23

While it's perfectly valid to include both `<inheritdoc/>` and other content, it only has an impact when the inherited documentation contains elements that are not manually replaced (e.g. if the inherited documentation had `<remarks>` or `<value>`, but this property only contained `<summary>`.